elasticsearch.md

```diff
@@ -309,6 +311,50 @@ readonlyrest:
 
 The keystore should be stored in the same directory with `elasticsearch.yml` and `readonlyrest.yml`.
 
+### Internode communication - transport module
+
+This option encrypts communication between nodes forming Elasticsearch cluster. 
+
+**⚠️IMPORTANT:** To enable SSL for internode communication open `elasticsearch.yml` and append this one line:
+
+```yml
+http.type: ror_ssl_internode
+```
+
+In `readonlyrest.yml` following settings must be added (it's just example configuration presenting most important properties):
+
+```yml
+readonlyrest:
+    ssl_internode:
+      keystore_file: "keystore.jks"
+      keystore_pass: readonlyrest
+      key_pass: readonlyrest
+```
+Similar to `ssl` for HTTP, keystore should be stored in the same directory with `elasticsearch.yml` and `readonlyrest.yml`.
+This config must be added to all nodes taking part in encrypted communication within cluster.  
+
+### Certificate verification 
+
+By default certificate verification is disabled. It means that certificate is not validated in any way, so all certificates are accepted.  
+It is useful on local/test environment, where security is not the most important concern. On production environment it is adviced to enable this option.
+It can be done by means of:
+
+```yml
+certificate_verification: true
+```
+under `ssl_internode` section. This option is applicable only for internode ssl.
+   
+### Client authentication 
+
+By default client authentication is disabled. When enabled, server asks client about its certificate, so ES is able to verify client's identity. It can be enabled by means of:
```

Comment:
By default, THE client authenticaiton is disabled. When enabled, THE server asks THE client about [...] ..so ES is able to verify THE client's identity.